assets/global.js

```diff
@@ -402,7 +422,8 @@ class MenuDrawer extends HTMLElement {
 
   onFocusOut(event) {
     setTimeout(() => {
-      if (this.mainDetailsToggle.hasAttribute('open') && !this.mainDetailsToggle.contains(document.activeElement)) this.closeMenuDrawer();
+      if (this.mainDetailsToggle.hasAttribute('open') && !this.mainDetailsToggle.contains(document.activeElement))
+        this.closeMenuDrawer();
```

Comment:
This is 139 chars long. Fits within 140 `printWidth`

Reply:
140 seemed a bit long when I was experimenting (had a few mega-long logic statements and CSS rules), but 120 seems to fit in well with existing line lengths.